File: Codeforces/707A - Brain's Photos.py
# ==================================================
# Problem  :   707A - Brain's Photos
# Run time :   0.077 sec.
# Language :   Python 3.5
# ==================================================

from sys import stdin, stdout
from re import search

def main():
	# stdin = open('input.txt', 'r')

	stdout.write("#Color\n" if search('C|M|Y', stdin.read()) else "#Black&White\n")

if __name__ == '__main__':
	main()

File: Codeforces/707A - Brain's Photos.c
// ==================================================
// Problem  :   707A - Brain's Photos
// Run time :   0.030 sec.
// Language :   C
// ==================================================

#include <stdio.h>

int main()
{
    int n, m, i, j;
    char clr, isColored;

    while(scanf("%d %d", &n, &m) != EOF) {
        isColored = 0;

        for(i = 0; i < n; ++i) {
            for(j = 0; j < m; ++j) {
                getchar();
                clr = getchar();

                if(!isColored && clr != 'B' && clr != 'W' && clr != 'G')
                    isColored = 1;
            }
        }

        puts((isColored)? "#Color" : "#Black&White");
    }

    return 0;
}
